### ISSUE TITLE
Unknown early instance "Composer\Autoload\ClassLoader"

### ISSUE THREAD

**Issue by infabo** [evoWeb/extender#22]
Once this extension is enabled, you can't flush caches from CLI anymore.

`typo3cms cache:flush` fails with:

![image](https://user-images.githubusercontent.com/3999104/64151433-a85b1300-ce2a-11e9-8655-14898e5ac576.png)

TYPO3 9.5.9
extender 6.6.0

**Comment by infabo** [comment]
https://github.com/evoWeb/extender/commit/2c4792d3ddf4b39e00f6d0ac2423e3d8d7071b39

Should be backported to 6.x

**Comment by lochmueller** [comment]
I got the same issue in our deployment `vendor/bin/typo3cms cache:flush` command. Vote up :)

**Comment by lochmueller** [comment]
Do you check the integration of backend.php `$classLoader = require __DIR__ . '/../../../../../../vendor/autoload.php';` or index.php. The loading of the autoload.php of composer looks much more cleaner.

**Comment by georgringer** [comment]
patch works for 6.x as well. @evoWeb  can you please backport that and release new version?

**Comment by garbast** [comment]
Applyed manualy 2c4792d and released as version 6.6.1

**Comment by garbast** [comment]
After fixing the tests the release is out now.

**Comment by infabo** [comment]
This is still broken in 6.6.1.

`vendor/bin/typo3cms 'cache:flush'`

**Comment by garbast** [comment]
Cant reproduce. Wont fix. If you feel the need to hand in a PR i will check if this breaks something else.

**Comment by infabo** [comment]
yeah, thanks

**Comment by BastianBalthasarBux** [comment]
Still broken in 6.6.2 in connection with typo3console, AFAICS.
Will investigate further.

**Comment by BastianBalthasarBux** [comment]
I read now through the suggested fixes, discussions etc.
Now I understand, what happens:
In "classic" non-composer based installations with symlinks, it is still broken. For example installing that extension breaks autoupdates on Mittwald servers, at least on that agency-server I have the fun to maintain now.

I suggest to add this line to the autoloaderFolders configuration in ClassCacheManager.php:

`PATH_site . 'typo3_src/vendor/',`

This will fix the problems for most people with the "classic" installation with symlinks, like it was suggested for ages, before composer appeared. ;)

**Comment by garbast** [comment]
Did ist work in the discribed environment or is it an assumption?

**Comment by BastianBalthasarBux** [comment]
It works, locally with "classic" mode, as well as on Mittwald Server. The only assumption: it will work for _most_ "classic" installations. (There may be some weird other installations outside, where also that will not work, but then they left the path of suggested ways to install TYPO3, e.g. without symlinks ... but even then, if I remember right, it would work).

To note: I used to use TYPO3 v9 on that systems, I did not test with v10, but I am pretty sure, that won't change anything.
